apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/(chainPage)/components/client/add-chain-to-wallet.tsx

```diff
@@ -12,58 +11,58 @@ import {
   useSwitchActiveWalletChain,
 } from "thirdweb/react";
 import { useDebounce } from "use-debounce";
+import { ToolTipLabel } from "../../../../../../../@/components/ui/tooltip";
+import { useCustomConnectModal } from "../../../../../../../@3rdweb-sdk/react/components/connect-wallet";
```

Comment:
use absolute paths here

Reply:
Added vscode setting to automatically do this